austrakka/utils/helpers/__init__.py

```diff
@@ -0,0 +1,5 @@
+from austrakka.utils.api import api_get
+
+
+def _get_by_identifier(path: str, identifier: str):
+    return api_get(path=f"{path}/{identifier}")['data']
```

Comment:
Is this one-line abstraction useful? Why not let api_get() unpack 'data'? And then get_analysis_by_abbrev() can just call api_get() directly.

Reply:
You're right; It's really not useful. I'll remove it